docs/notebooks/usage.ipynb

```diff
@@ -26,14 +26,14 @@
     "import xarray as xr\n",
```

Comment:
<p>Singular of <code>indices </code>  is <code>index</code> I believe </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xclim/pull/1338/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
This is something I've always wondered about: I deliberately have never corrected `indice` to `index` because I found it to be confusing, given that we do so much "indexing" in the code. There's more than 15 places in the docs where we use `indice` instead of `index`. Should we correct that?

Reply:
For the moment, it's changed to `index`, but if you come across a rationale for changing it (could `indice` be a British English term??), let me know via Slack!